packages/cli/src/commands/generate/function/__tests__/__snapshots__/function.test.ts.snap

```diff
@@ -121,6 +132,10 @@ export const handler = async (event, context) => {
 
   return {
     statusCode: 200,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+
```

Comment:
Let's remove this newline to be consistent with formatting.

Or are these snaps auto-generated?

Reply:
@dthyresson  Yes, they were auto-generated: I ran `yarn test -u`, which updated 4 snapshots automatically (which I wouldn't have known where to find or how to update). Thanks.